debug.log

```diff
@@ -1 +1,2 @@
 [1124/122051.086:ERROR:crash_report_database_win.cc(457)] unexpected header
+[1208/112014.877:ERROR:crash_report_database_win.cc(455)] unexpected header
```

Comment:
It is better not to have the log file in the repository. Please use .gitignore for it.

Reply:
What is a .gitignore ?

Reply:
Done